tools/tests/colon-converter-tests.js

```diff
@@ -77,7 +77,7 @@ if (process.platform !== "win32") {
     });
   });
 
-  selftest.define("can't build local packages with colons", function () {
```

Comment:
I'm really surprised to hear that this test is flaky; it doesn't seem to communicate with any external servers. What sort of flakiness were you seeing?